tomviz/CMakeLists.txt

```diff
@@ -512,6 +514,22 @@ foreach(file ${resource_files})
     COMPONENT runtime)
 endforeach()
 
+set(template_files
+  Back_Projection_Reconstruction.tvsm
+)
+
+# Install templates
+file(MAKE_DIRECTORY "${tomviz_BINARY_DIR}/share/tomviz/Templates")
```

Comment:
Let's keep `templates` lower case for all paths please.

Reply:
Fixed.